### ISSUE TITLE
A11y_Azure Tools for VS Code_Azure Function_Collapse All_Usability: keyboard focus is landing on non-interactive (Collapsed All) control.

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode-azurefunctions#3818]
### GitHubTags:
#A11yUsable; #A11yTCS; #A11ySev4; #Azure Tools for VS Code; #BM-AzureToolforVSCode-MAC-AUG2023; #DesktopApp;  #AINotInScope; #Device; #MAC;

### OS Details:
Mac OS Ventura 13.5

### Environment Details:
Application Name: Azure Tools for VS Code

### Repro Steps:
1.Open Visual studio code
2. Tab till extension button and search for "Azure Functions"
3. Tab till install and navigate to the "Azure" button.
4. Tab till "Collapse All Button" from the toolbar in "workspace" section.
5.Verify whether keyboard focus is landing on non-interactive (Collapsed All) control or not.

### Actual Result:
keyboard focus is landing on non-interactive (Collapsed All) control.

### Expected Result:
keyboard focus should not lands on non-interactive (Collapsed All) control.

### User Impact:
keyboard user will perform unnecessary actions as focus is going on non interactive controls.

### Attachment:
![Non interactive](https://github.com/microsoft/vscode-azurefunctions/assets/98530069/e9b200e3-5d2b-43b3-aca3-90fdef45c463)

**Comment by MicroFish91** [comment]
Hi @kupatkar99, I believe this is a default VS Code behavior that will reproduce with any extension that register's a tree view using VS Code's internal API (but does not have any tree nodes to display yet).  As such, the automatic contribution and behavior of that icon would ultimately be dictated by VS Code and not the individual extension.  

Unless you have reason to believe that our extension is specifically the cause of this issue, would you be open to opening this issue with [VS Code](https://github.com/microsoft/vscode/issues) instead as they are better equipped to figure out how this needs to be addressed?  Thank you!

**Comment by MicroFish91** [comment]
Tracking VS Code issue: https://github.com/microsoft/vscode/issues/263684

**Comment by meganrogge** [comment]
This is useful to still have the focus move to the action so screen reader users are aware of it.

**Comment by MicroFish91** [comment]
@kupatkar99 Hi, please see upstream VS Code response.  Could we have this issue closed out? Thank you
